### PR TITLE
Issue #8192: Translate missing zip file msg in com.ibm.ws.artifact.zip

### LINKED ISSUE #8192: Untranslated library error message
ZipFileContainerFactory.java contains an untranslated error message.  
      } catch ( FileNotFoundException e ) {
            Tr.error(tc, "Missing zip file " + file.getAbsolutePath());
            return false;
You can easily duplicate this failure by creating a library like this
    	<jdbcDriver id="OracleDriver">
    	    <library name="OracleLib">
    	    	<fileset dir="/Users/user/Dev/jdbc_drivers/oracle-ojdbc6-debug/" includes="ojdbc6_g.jar"></fileset>
    		</library>    	
    	</jdbcDriver>

and removing all perms from the lib file.
ls -l ojdbc6_g.jar 
----------  1 user  staff  5865821 Nov  4  2016 ojdbc6_g.jar

  This appears in console.log of Liberty 
[ERROR   ] Missing zip file /Users/user/Dev/jdbc_drivers/oracle-ojdbc6-debug/ojdbc6_g.jar

### PR DESCRIPTION
**Pull request by hlhoots**
#build 
fixes #8192

### CODE REVIEW

**Review by jimblye — APPROVED** [review]

**Comment by hlhoots** [comment]
@dmuelle  or @chirp1 can I get a message review on this one.  Thanks!

File: dev/com.ibm.ws.artifact.zip/resources/com/ibm/ws/artifact/zip/internal/resources/ZipArtifactMessages.nlsprops
Comment (dmuelle) [inline dev/com.ibm.ws.artifact.zip/resources/com/ibm/ws/artifact/zip/internal/resources/ZipArtifactMessages.nlsprops]: per ibm style, recommend to put the {variable} before the identifier, like 

`CWWKM0128E: The {0} zip file is missing`

File: dev/com.ibm.ws.artifact.zip/resources/com/ibm/ws/artifact/zip/internal/resources/ZipArtifactMessages.nlsprops
Comment (dmuelle) [inline dev/com.ibm.ws.artifact.zip/resources/com/ibm/ws/artifact/zip/internal/resources/ZipArtifactMessages.nlsprops]: for clarity in translation, avoid elliptical clauses such as "while processing". also avoid parentheses except to specify an acronym or term. what is the exception called? If it's `notFound` or similar could be something like:

`A notFound exception occurred during archive processing.`
or
`A notFound exception occurred when the archive was processed.`
or
`An exception occurred during archive processing because the specified file was not found`

**Review by dmuelle — COMMENTED** [review]
couple minor things- let me know if you have any questions. Thanks!

**Comment by hlhoots** [comment]
@dmuelle Thank you!  Updates made based on your suggestions.  Let me know if those changes look good.

#build

**Comment by hlhoots** [comment]
Build had 2 unrelated UI failures.  Should be good to merge once ID approval is completed.

**Review by dmuelle — APPROVED** [review]
LGTM thanks

**Comment by hlhoots** [comment]
#run-libby-bot

**Pull request merged**
